bin/secrets/metadata.sh

```diff
@@ -36,3 +36,8 @@ nodes_map["secret/jenkins/tests/practitest/"]="$slaves"
 paths_map["secret/jenkins/tests/practitest/config.ini"]="/home/jenkins-slave/.practitest/config.ini"
 before_scripts_map["secret/jenkins/tests/practitest/"]="rm -rf /home/jenkins-slave/.practitest && mkdir -p /home/jenkins-slave/.practitest"
 after_scripts_map["secret/jenkins/tests/practitest/"]="chown -R jenkins-slave:jenkins-slave /home/jenkins-slave/.practitest"
+
+nodes_map["secret/jenkins/tests/snapcraft/"]="$slaves"
+paths_map["secret/jenkins/tests/snapcraft/snapcraft.cfg"]="/home/jenkins-slave/.config/snapcraft/snapcraft.cfg"
```

Comment:
Now that #134 has landed remember to upload this secret encoded, I think that the easiest way to do it is putting the file in a path like `~/secrets_backup/secret/jenkins/tests/snapcraft/snapcraft.cfg` and then calling `./bin/secrets/restore.sh ~/secrets_backup/`